### PR TITLE
Issue #SB-30214 fix: create content API issue fixes

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: src/app/editors/contentlist/contentlist.component.spec.ts [hunk 1]
```diff
@@ -79,12 +79,12 @@ describe('ContentlistComponent', () => {
     const helperService: HelperService = TestBed.inject(HelperService);
     const createContentRes = {
       result: {
-        identifier: 'do_1233'
+        content_id: 'do_1233'
```
Comment (vinukumar-vs) [inline src/app/editors/contentlist/contentlist.component.spec.ts:82]: Why are changing this. "identifier" was the right one that you get in the response. This should be also checked for "identifier".

File: src/app/editors/contentlist/contentlist.component.ts [hunk 2]
```diff
@@ -81,8 +81,7 @@ export class ContentlistComponent implements OnInit {
     }
    };
     this.helperService.createContent(requestData).subscribe(res => {
-      console.log(res.result.identifier);
-      this.openContent(res.result.identifier);
+      this.openContent(res.result.content_id);
```
Comment (vinukumar-vs) [inline src/app/editors/contentlist/contentlist.component.ts:84]: "identifier" was right. Why did you changed now?

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: src/app/editors/contentlist/contentlist.component.ts (hunk 2 above)
Comment (vaibhavbhuva) [reply src/app/editors/contentlist/contentlist.component.ts:84]: @vinukumar-vs After changing the API, We are getting `content_id` in the response.

File: src/app/editors/contentlist/contentlist.component.spec.ts (hunk 1 above)
Comment (vaibhavbhuva) [reply src/app/editors/contentlist/contentlist.component.spec.ts:82]: @vinukumar-vs After changing the API, We are getting content_id in the response.

**Pull request merged**